src/Manager.cpp

```diff
@@ -142,5 +142,20 @@ namespace ClassProject
 
     }
 
+    /**
+    * @brief getTopVarName
+    *
+    * returns the name of the top variable of the given node.
```

Comment:
I think this function returns the name of the node with the given id, not the name of the top variable of the given node... Is it what you want to achieve?